.vscode/settings.json

```diff
@@ -1,7 +1,13 @@
 {
-	"editor.defaultFormatter": "rvest.vs-code-prettier-eslint",
-	"editor.formatOnPaste": false, // required for vs-code-prettier-eslint
-	"editor.formatOnSave": true, // required for vs-code-prettier-eslint
-	"editor.formatOnType": false, // required for vs-code-prettier-eslint
-	"editor.formatOnSaveMode": "file" // required to format on save for vs-code-prettier-eslint
-}
```

Comment:
you removed the tab indents :(

Reply:
Last try, I will explicitly set my global settings to tabs.

Also, although it's been receiving hate lately, I think pre-commit hooks are very good and useful. If a dev wants to disable, they'll be caught in CI. What do you think?

Reply:
@coyotte508 I'm using the vscode builtin `JSON Language Features` which automatically formats JSON files. Is that what you use? If not can you share what you're using and if you have any custom settings? All these tabs to space changes from my commits seem to be automatically happening from my editor.

![CleanShot 2023-03-09 at 11 04 09](https://user-images.githubusercontent.com/23618431/224128622-278ef978-23e3-4a3e-92ac-26e588b33b7f.png)

Reply:
> Also, although it's been receiving hate lately, I think pre-commit hooks are very good and useful. If a dev wants to disable, they'll be caught in CI. What do you think?

#1 :smiling_face_with_tear: 

I have no strong opinion, @pngwn seems firmly against it though. We can maybe discuss a bit later when there's more contributors.

Still, with the latests changes in main, it should be fixed by `pnpm -r format`